### ISSUE TITLE
Is there an event source for System.Net.Sockets

### ISSUE THREAD

**Issue by Malcolm-Stewart** [microsoft/perfview#1701]
Similar to this article https://docs.microsoft.com/en-us/dotnet/framework/network-programming/how-to-configure-network-tracing

**Comment by brianrob** [comment]
Yes - for .NET Core, there is an eventsource: https://github.com/dotnet/runtime/blob/13733cdd969842de9c36235d730a604ce55ee729/src/libraries/System.Net.Sockets/src/System/Net/Sockets/NetEventSource.Sockets.cs#L10.
